README.md

```diff
@@ -32,6 +32,8 @@ npm run bootstrap:ci
 > - Installs dependencies for the [libraries](#components).
 > - Builds all the [libraries](#components).
 
+Read [Install Kyma Dashboard manually](docs/install_kyma_dashboard_manually.md) to learn how to install the Dashboard with Istio Ingress and how to install it on a Kyma cluster.
```

Comment:
Unfortunately, we also have [guidelines](https://kyma-project.io/community/guidelines/technical-guidelines/01-naming/) to use `-` instead of `_` in filenames. Could you correct it as well?